### PR TITLE
Run system tests against multiple RabbitMQ versions

### PR DESCRIPTION
**Pull request by ansd**

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -46,12 +46,17 @@ jobs:
     strategy:
       matrix:
         k8s: [v1.17.11, v1.18.8, v1.19.4, v1.20.0]
+        rabbitmq-image:
+        - rabbitmq:3.8.8-management
+        - rabbitmq:3.8.9-management
+        - rabbitmq:3.8.10-rc.5-management
```
Comment (ChunyiLyu) [inline .github/workflows/pr.yml:52]: `3.8-rc-management` could be a better tag because there might be `rc.6` in the future.

**Review by Zerpet — APPROVED** [review]

**Review by ChunyiLyu — CHANGES_REQUESTED** [review]
Small thing, but please see comment 👍

**Comment by ChunyiLyu** [comment]
@ansd  I don't know what happened with my comment. I was suggesting to use the tag `3.8-rc-management` to test rcs instead of a specific rc version.

**Review by ChunyiLyu — APPROVED** [review]
yayy 😄

**Pull request merged**